### DISCUSSION TITLE
Netbeans stuck on 'Activating PHP'

### DISCUSSION THREAD

**Discussion by kuba14** [apache/netbeans#6664] [Q&A] [answered]
I have Netbeans IDE 19 installed. When I open it and open my PHP project, Netbeans is stuck in "Activating PHP" state with no other information. Can anybody help with this?

**Comment by rsantellan** [comment]
@kuba14 I started from the console and it was fixed, also I download it from here https://www.codelerity.com/netbeans/ I'm on Debian.

**Reply by kuba14** [reply]
How did you started it from the terminal? I have installed Netbeans now and when I run it from Desktop, the issue is still there.

**Comment by rsantellan** [comment, marked as answer]
@kuba14 just type:
`netbeans`
And don't close the terminal window

**Reply by kuba14** [reply]
It works, thanks!

**Reply by Chris2011** [reply]
also please try to use the newest NetBeans 24 version.

**Comment by Anderseta** [comment]
This happened to me when i tried to open an old netbeans php project (version 11) with Netbeans 24.
To fix this, start a new project with existing source code
